18 Normalizr/src/schemas/countrySchema.ts

```diff
@@ -0,0 +1,4 @@
+import { schema } from 'normalizr';
+
+export const countrySchema = new schema.Entity('countries');
```

Comment:
I think we should put all schemas on the same file, so you can export without the schema prefix, like `mySchema.country`.